### ISSUE TITLE
Create Database

### ISSUE THREAD

**Issue by FendVario1** [amosproj/amos2022ss01-firmware-downloader#33]
## User story
1. As a Developer
2. I want / need a working database environment
3. So that data of the downloader can be persisted.

## Acceptance criteria
* Database (e.g. SQLite) is started and initialized on program startup
* Data is persistently stored

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by TejeshPala** [comment]
The Development of build for Initial Database model is complete
![Data Base Config,Setup](https://user-images.githubusercontent.com/104498119/170153562-d8c8e597-ae3e-4f8c-b033-1180cd54967b.png)
d

**Comment by TejeshPala** [comment]
The Code is merged into Development branch with name database.py

https://github.com/amosproj/amos2022ss01-firmware-downloader/blob/TejeshPala/database.py - Code
https://github.com/amosproj/amos2022ss01-firmware-downloader/blob/TejeshPala/firmwaredatabase.db - database sample file
https://github.com/amosproj/amos2022ss01-firmware-downloader/blob/TejeshPala/Data%20Base%20Config%2CSetup.xlsb - Config of DB and setting in DB
